airbyte-webapp/src/test-utils/mock-data/mockConnection.ts

```diff
@@ -1,4 +1,4 @@
-/* eslint-disable no-template-curly-in-string */
+/* eslint-disable no-template-curly-in-string, @airbyte/no-hardcoded-connector-ids */
```

Comment:
Why disable the rule here instead of using the ConnectorIds object?

Reply:
My original thought was, that these are just test mocks, we're really not caring that those are real IDs in there. That said there's also absolutely not reason NOT to use it here, so I've changed it.